.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
     "**/.yarn": true,
     "**/.pnp.*": true
   },
-  "prettier.prettierPath": "./webapp/.yarn/sdks/prettier/index.js"
+  "prettier.prettierPath": "./webapp/.yarn/sdks/prettier/index.js",
+  "eslint.workingDirectories": [{ "mode": "auto" }]
```

Comment:
general knowledge

Why this?

Reply:
This specifies how a directory works, on [eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) explain better